### PR TITLE
Use CENNZnetEventProof struct to tidy up args

### PR DESCRIPTION
**Pull request by jordy25519**
Include validatorSetId in proof to allow verifying older events

### CODE REVIEW

**Review by KarishmaBothara — APPROVED** [review]

**Comment by KarishmaBothara** [comment]
I get few errors compiling the contract.. will try to fix these

File: crml/erc20-peg/contracts/CENNZnetBridge.sol [hunk 1]
```diff
@@ -85,14 +85,14 @@ contract CENNZnetBridge is Ownable {
         address[] memory newValidators,
         CENNZnetEventProof memory proof
     ) external {
-        require(validatorSetId > validatorSetId, "validator set id replayed");
+        //require(validatorSetId > validatorSetId, "validator set id replayed");
```
Comment (jordy25519) [inline crml/erc20-peg/contracts/CENNZnetBridge.sol]: `proof.validatorSetId > activeValidatorSet`

File: crml/erc20-peg/contracts/ERC20Peg.sol [hunk 2]
```diff
@@ -44,7 +44,7 @@ contract ERC20Peg is Ownable {
     // i.e. v[i], r[i], s[i] matches the i-th validator[i]
     function withdraw(address tokenType, uint256 amount, CENNZnetEventProof memory proof) payable external {
         require(withdrawalsActive, "withdrawals paused");
-        bytes32 message = abi.encodePacked(tokenType, amount, msg.sender);
+        bytes32 message = keccak256(abi.encodePacked(tokenType, amount, msg.sender));
```
Comment (jordy25519) [inline crml/erc20-peg/contracts/ERC20Peg.sol]:
```suggestion
        bytes32 message = abi.encodePacked(tokenType, amount, msg.sender)
```

Comment (jordy25519) [reply crml/erc20-peg/contracts/ERC20Peg.sol]: leave the message hashing to the bridge contract

Comment (KarishmaBothara) [reply crml/erc20-peg/contracts/ERC20Peg.sol]: Screen Shot 2021-08-30 at 1 49 53 PM
If I don't use keccak256, then it fails to compile

Comment (jordy25519) [reply crml/erc20-peg/contracts/ERC20Peg.sol]: ah should be `bytes` not `bytes32`

Comment (KarishmaBothara) [reply crml/erc20-peg/contracts/ERC20Peg.sol]: done... ✅ 
used 
`byte memory message`

**Comment by jordy25519** [comment]
moved to cennznet/bridge-contracts

**Pull request closed without merging**